ci/integration_tests/configs/local.yaml

```
@@ -0,0 +1,17 @@
+chain_cfg: 
+  denom: "ujunox"
+  prefix: "juno"
+  chain_id: "testing"
+  rpc_endpoint: "http://localhost:26657/"
+  gas_prices: 0.1
+  gas_adjustment: 1.5
+
+# Add code ids if you dont need to store them:
+code_ids:
+  "cw20_base": 1
+  "cw4_group": 2
+  "cw_core": 8
+  "cw_proposal_single": 13
+  "cw4_voting": 5
+  "cw20_staked_balance_voting": 4
+  "cw20_stake": 3
```

When running this locally I get these:
```suggestion
code_ids:
  "cw20_base": 1
  "cw4_group": 2
  "cw_core": 9
  "cw_proposal_single": 14
  "cw4_voting": 6
  "cw20_staked_balance_voting": 5
  "cw20_stake": 4

---

Could we also perhaps add a section in the README about how to keep this up to date? I foresee some CI failures when people modify the deploy local script.

---

So, this wont break the CI, because the CI uses [`configs/ci.yaml`](https://github.com/DA0-DA0/dao-contracts/blob/39903678b8fdf7a74f8f5754512c4eb37aec269b/ci/integration_tests/configs/ci.yaml) which doesn't have any contract ids stored. It will always just store all of the contracts from a fresh slate before all of the tests run.

I will make the local usage options clearer in the README though.